src/main/java/core/basesyntax/serviceimpl/CalculateValues.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.strategyimpl.StrategyImpl;
+import java.util.List;
+
+public class CalculateValues {
+    public void getFruitsAndValues(List<FruitTransaction> info) {
+        StrategyImpl strategy = new StrategyImpl();
```

Comment:
Are you sure that it is good idea to create this object each time method is called?